frontend/device/sdl/device.lua

```diff
@@ -144,7 +145,12 @@ function Device:init()
     end
 
     self.hasClipboard = yes
-    self.screen = require("ffi/framebuffer_SDL2_0"):new{device = self, debug = logger.dbg}
+    self.screen = require("ffi/framebuffer_SDL2_0"):new{device = self, debug = logger.dbg,
```

Comment:
Probably clearer to stick them all on a separate line. In my five-minute hack I didn't bother with formatting. ;-)